src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -391,19 +391,19 @@ default String checkSelectedOption(String expectedText, double waitFor, double t
      * @return String: the actual selected value of the element. null will be returned if the element isn't present or a select
      */
     @SuppressWarnings("squid:S1168")
-    default String checkSelectedValue(String expectedValue, double waitFor, double timeTook) {
+    String checkSelectedValue(String expectedValue, double waitFor, double timeTook) {
         // record the action and check it's a select
         if (!isPresentSelect(
-                getElement().prettyOutput() + " having a selected value of <b>" + expectedValue + "</b>", waitFor)) {
+                this.element.prettyOutput() + " having a selected value of <b>" + expectedValue + "</b>", waitFor)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " having a selected value of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)